src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -70,117 +68,76 @@ const AdminDashboard = () => {
     window.scrollTo(0, 0);
   }, [HAS_VISITED_BEFORE]);
 
+  // resets the selected incidents when switching from approved to unapproved page
+  useEffect(() => {
+    setSelectedIds([]);
+    setAllSelected(false);
+  }, [listType]);
+
   const modalHandler = () => {
     setShowModal(false);
   };
 
-  const lastPage = Math.ceil(unapprovedIncidents.length / incidentsPerPage);
-
   const oktaAxios = useOktaAxios();
 
-  // getting unapproved/pending incidents from the database
-  useEffect(() => {
-    getData(oktaAxios, setUnapprovedIncidents);
-  }, []);
+  const fetchIncidents = () => {
+    getApprovedIncidents(oktaAxios)
+      .then(res => setApprovedIncidents(res))
+      .catch(err => console.log(err));
 
-  // getting approved incidents from database
-  useEffect(() => {
-    oktaAxios
-      .get('/dashboard/incidents/approved')
-      .then(res => {
-        setIncidents(res.data);
-      })
```

Comment:
So this useEffect was to load up the approved list view via that axios request, is this just being done elsewhere now?

Reply:
Yes, originally there were three separate useEffect hooks for pending/approved/form incidents, and you could tell they were written at different times because they were each different. They've been rewritten and combined into a single function, and called from a single useEffect();